tests/test_callback_best_epoch.py

```diff
@@ -16,18 +17,29 @@ def pipeline_kwargs() -> Dict[str, Any]:
 
 
 @pytest.fixture(scope="module")
-def dataset_kwargs() -> Dict[str, Any]:
-    return {
-        "dataset_name_or_path": "clarin-pl/polemo2-official",
-        "input_column_name": ["text"],
-        "target_column_name": "target",
-        "load_dataset_kwargs": {
+def dataset_kwargs() -> Tuple[Dict[str, Any], "TemporaryDirectory[str]"]:
+    path = TemporaryDirectory()
+    pipeline = HuggingFacePreprocessingPipeline(
+        dataset_name="clarin-pl/polemo2-official",
+        load_dataset_kwargs={
             "train_domains": ["hotels", "medicine"],
             "dev_domains": ["hotels", "medicine"],
             "test_domains": ["hotels", "medicine"],
             "text_cfg": "text",
         },
-    }
+        persist_path=path.name,
+        sample_missing_splits=None,
+        ignore_test_subset=False,
+        downsample_splits=(0.01, 0.01, 0.05),
+        seed=441,
+    )
+    pipeline.run()
+
+    return {
+        "dataset_name_or_path": path.name,
+        "input_column_name": ["text"],
```

Comment:
I would expect single "text" because of name suffix and I wouldn't expect list -> consider adding s as suffix so that it indicates plural names

Reply:
Naming of this variable is problematic, as for unification we introduced one variable that stores input_column_name in pipelines. It can be single value or list/tuple. What do you think we should do with that?

Reply:
plural names allow for list or tuple with single value inside, that's fine and singular name with list or tuple seems worse option imo - I'd go with input_column_name -> input_column_names

Reply:
Added to #120 This naming problem is above scope of this PR as it occurs also in other pipelines (like lighting classification). Currently name is same as in other places